src/Adapter/Image/ImageRetriever.php

```diff
@@ -190,6 +199,48 @@ public function getImage($object, $id_image)
                 );
             }
 
+            $generateAdditionalWebP = (bool) Configuration::get('PS_ADDITIONAL_IMAGE_WEBP');
+
+            if ($generateAdditionalWebP) {
+                $resizedImagePathWebP = implode(DIRECTORY_SEPARATOR, [
+                    $imageFolderPath,
+                    $id_image . '-' . $image_type['name'] . '.webp',
```

Comment:
It's usually a best practice to use sprintf rather than concatenating when there's more than one variable, it's also more readable:

```suggestion
                    sprintf('%s-%s.webp', $id_image, $image_type),
```